rust/polars-ocaml/src/utils.rs

```diff
@@ -25,6 +25,8 @@ pub unsafe fn ocaml_invalid_argument(error_message: &str) -> ! {
     unreachable!("caml_failwith should never return")
 }
 
+polars_ocaml_macros::ocaml_interop_backtrace_support!();
```

Comment:
Why use macros rather than writing the code here?

Reply:
> Why use macros rather than writing the code here?
It felt weird to me to have code in a macro (`raise_ocaml_exception_from_panic` referenced from `ocaml_interop_export`) that implicitly assumes some implementation in the `polars-ocaml` crate, so wanted the code to be in the same place. I considered having a function exported from the macro crate, but quickly realized that that's not allowed.

I see how it would be cleaner to extract out the runtime into a separate crate, though. I'll add a TODO to address this if we ever decide to extend the functionality here or make it any more complex. How does that sound?

Reply:
For example, serde-derive generates code assuming there is a crate called serde with appropriate implementation.

One thing you can do is to declare the function in a #[doc(hidden)] module so that it doesn't appear on docs (so normal users aren't aware of it)